src/wrappers/ContentUploader.js

```diff
@@ -40,6 +40,16 @@ class ContentUploader extends ES6Wrapper {
         this.emit('error', data);
     };
 
+    /**
+     * Callback when a file uploads that gets called in the t3 page for Upload Embed Widget
```

Comment:
Change the comment to suggest what the event is doing instead of what its for, since this is public api